pkg/statistics/handle/types/interfaces.go

```diff
@@ -115,6 +115,15 @@ type StatsAnalyze interface {
 	// DeleteAnalyzeJobs deletes the analyze jobs whose update time is earlier than updateTime.
 	DeleteAnalyzeJobs(updateTime time.Time) error
 
+	// CleanupCorruptedAnalyzeJobsOnCurrentInstance cleans up the corrupted analyze job.
+	// We use current running analyze jobs to check whether the analyze job is corrupted.
+	CleanupCorruptedAnalyzeJobsOnCurrentInstance(currentRunningProcessIDs map[uint64]struct{}) error
```

Comment:
Suggest that we add a simple explanation of what "corrupted" means since it's a new term in the code.
Also in the comments for `analyzeJobsCleanupWorker`.

Reply:
Updated.